CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -10,23 +10,27 @@ import UIKit
 
 class OnboardingTableViewCell: UITableViewCell {
 
+    private enum LastShownItem {
+        case interest, group, none
+    }
+
     // MARK: Private View Vars
     private let backdropView = UIView()
     private let interestImageView = UIImageView()
     private let titleLabel = UILabel()
-    private lazy var categoriesLabel: UILabel = {
-        let categoriesLabel = UILabel()
-        categoriesLabel.textColor = .greenGray
-        categoriesLabel.font = ._12CircularStdBook
-        contentView.addSubview(categoriesLabel)
-        return categoriesLabel
-    }()
-
-    // True if the cell has layed out views at least once
-    private var initialized = false
-    // Store whether it was showing interests or groups so it doesn't relayout on dequeue
-    private var showingInterests = false
-    // Whether the cell should change its appearence when setSelected is called
+    private let categoriesLabel = UILabel()
```

Comment:
a-z. Granted, I don't care about alphabetization, but it appears to be AppDev standards. We should have a script of sorts that arranges groups of variables and imports alphabetically.